### PR TITLE
:rocket:  Release v1.9.0

### PR DESCRIPTION
**Pull request by peppi-lotta**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: releasenotes/v1.9.0.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+- fix dependabot config ignore rules (#672)
+- add dependabot config for release-1.8 (#657)
+
+## :recycle: Superseded or Reverted
+
+(#661)(#663)(#654)(#666)(#669)(#681)(#682)(#688)(#698)(#704)(#708)(#711)(#712)
+(#714)(#718)(#725)(#731)(#732)(#735)(#744)(#751)(#755)(#761)
+
+
+- `<insert superseded bumps and reverts here>`
```
Comment (mquhuy) [inline releasenotes/v1.9.0.md:58]: Remove this line.

**Review by mquhuy — COMMENTED** [review]
A nit. Otherwise lgtm

**Review by lentzi90 — COMMENTED** [review]
/approve

**Pull request merged**